hieradata/common.yaml

```diff
@@ -1588,7 +1588,9 @@ router::nginx::robotstxt: |
   Disallow: /
   # Complaints of 429 'Too many requests' seem to be coming from SharePoint servers
   # (https://social.msdn.microsoft.com/Forums/en-US/3ea268ed-58a6-4166-ab40-d3f4fc55fef4)
```

Comment:
Do we still need this comment?

Reply:
I think so - it's still that bot, but both links combine to show the actual issue.